### DISCUSSION TITLE
Welcome to macnotesapp Discussions!

### DISCUSSION THREAD

**Discussion by RhetTbull** [RhetTbull/macnotesapp#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by josca42** [comment]
Hey.

I implement machine learning systems and have done so for the last 7 years. As so many other I am currently playing around with chatGPT and other large language modes (it really is astounding how well they work). 
As a side project I'm currently creating my own personal assistant. I'm giving chatGPT access to my emails, calendar and apple notes. 

After having tried countless note taking apps I always end back up using apple notes. Really excited to integrate chatGPT and apple notes.

And thanks for making this project. By far the best project I found to interact with apple notes.

**Reply by RhetTbull** [reply]
Thanks for the note! Sounds like a fascinating project! 

I too have tried several different notes apps, was an Evernote user for over a decade, but keep coming back to Apple notes for its simplicity and ubiquity. I just wish it was easier to programmatically access my data, hence this project.  Please open an issue if you run into any problems or have ideas for new features.
